Library/SRRecorderControl.m

```diff
@@ -422,6 +430,11 @@ - (NSString *)accessibilityLabel
     return label;
 }
 
+- (NSString *)accessibilityTitle
```

Comment:
Does this duplication resolve a specific issue?

Reply:
(label is intended to be a clean replacement for title, but there have definitely been bugs historically)